src/state.rs

```diff
@@ -303,8 +287,8 @@ mod tests {
     #[test]
     fn initial_config() {
         let config = mock_config(4);
-        let state0 = PbftState::new(0, &config);
-        let state1 = PbftState::new(1, &config);
+        let state0 = PbftState::new(vec![0], &config);
+        let state1 = PbftState::new(vec![], &config);
```

Comment:
Should this be `vec![1]`?

Reply:
It shouldn't matter, it never really gets used.